### ISSUE TITLE
Partial AOI Download, no error message or details provided

### ISSUE THREAD

**Issue by district1130** [lmotta/layertilesmapcanvas#2]
After generating the fishnet, and completing the download, it seems to always miss at least a few, and sometimes a majority of the requested tiles.  I don't know if this is a tile server issue, or a program issue, but is there a way to tell if the plugin tried and failed to retrieve a given tile?

**Comment by lmotta** [comment]
Please, show me the XYZ server that you are using for tiles download.

**Comment by district1130** [comment]
https://cartocdn-gusc.global.ssl.fastly.net/pablo-cartodb/api/v1/map/pablo-cartodb@e3787a3e@447f40d49e23cde521b720d310aa2b09:1552477703787/1,2/{z}/{x}/{y}.png

**Comment by district1130** [comment]
I tried several different zoom levels and AOIs, and sthe issue occured to some degree in all the cases.  When I load the XYZ tiles normally, the map renders without any holes in it.  I ran into this issue on both macOS and Windows.

**Comment by lmotta** [comment]
I use Linux/Ubuntu (20.04) and QGIS 3.16.3.
With yours XYZ server, I did all download of tiles, TIF files.
The problem found was with VRT option, where, the VRT image(mosaic) is wrong, but, without VRT option, the tiles images was correct.

About your question, if has error when do download, the plugin emit a warning message to the log error.

**Comment by district1130** [comment]
I will try it again on Ubuntu, and hopefully it will work there. Thanks for
checking it out

**Comment by lmotta** [comment]
![Peek 22-01-2021 20-11](https://user-images.githubusercontent.com/193559/105558762-1806c200-5cee-11eb-990d-83bb26420075.gif)

**Comment by district1130** [comment]
When using Ubuntu, the error still occurs, but only when downloading large datasets.  I attempted to download a tileset that was about 8000 total tiles, and only 950 downloaded before the plugin stopped downloading tiles.  I tried again, on a different Ubuntu machine, in a different area of the world, again attempting to download about 8000 tiles, and I only made it to 966 before it broke the second time.  

The plugin didn't report an error, it just continued to show progress on downloading tiles, when none were actually being downloaded.
